docs/whats-new/2019-12.md

```diff
@@ -0,0 +1,136 @@
+- [Tutorial: Debug a memory leak in .NET Core](../core/diagnostics/debug-memory-leak.md) - Diagnostic mem leak tutorial
+- [How to install localized IntelliSense files for .NET Core](../core/install/localized-intellisense.md) - Install localized IntelliSense
+- [How to install localized IntelliSense files for .NET Core](../core/install/localized-intellisense.md) - Install localized IntelliSense
+- [Run-time configuration options for compilation](../core/run-time-config/compilation.md) - Run-time config knobs
+- [Run-time configuration options for debugging and profiling](../core/run-time-config/debugging-profiling.md) - Run-time config knobs
+- [Run-time configuration options for globalization](../core/run-time-config/globalization.md) - Run-time config knobs
+- [Run-time configuration options for networking](../core/run-time-config/networking.md) - Run-time config knobs
+- [Run-time configuration options for threading](../core/run-time-config/threading.md) - Run-time config knobs
+- [Get started with .NET Core on Windows/Linux/macOS using the command line](../core/tutorials/cli-create-console-app.md) - Update cli tutorial for 3.1
+- [What's new in .NET Core 3.1](../core/whats-new/dotnet-core-3-1.md) - Update multiple docs for .NET Core 3.1
+
+## C# language
+
+### New articles
+
+- [Enumeration types (C# reference)](../csharp/language-reference/builtin-types/enum.md) - Moved and revised the enum article
+- [Learn about any breaking changes in the C# compiler](../csharp/whats-new/breaking-changes.md) - Add links to breaking changes doc
+
+### Updated articles
+
+- [How to join two collections (LINQ to XML) (C#)](../csharp/programming-guide/concepts/linq/how-to-join-two-collections-linq-to-xml.md) - Acrolinx 12/17/19
+- [How to list all nodes in a tree (C#)](../csharp/programming-guide/concepts/linq/how-to-list-all-nodes-in-a-tree.md) - Acrolinx 12/17/19
+
+## .NET framework
+
+### Updated articles
+
+- [\<AppContextSwitchOverrides> element](../framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md) - Acrolinx 12/20
+- [Security overview](../framework/data/adonet/security-overview.md) - Acrolinx 12/26
+- [Marshaling Classes, Structures, and Unions](../framework/interop/marshaling-classes-structures-and-unions.md) - Add links to table
+- [Security exceptions](../framework/wcf/diagnostics/exceptions-reference/security-exceptions.md) - Acrolinx 12/23
+
+## .NET for Apache Spark
+
+### New articles
+
+- [Submit a .NET for Apache Spark job to Databricks](../spark/how-to-guides/databricks-deploy-methods.md) - Spark: how to deploy to databricks
+- [Tutorial: Do batch processing with .NET for Apache Spark](../spark/tutorials/batch-processing.md) - Spark: batch processing
+- [Tutorial: Structured Streaming with .NET for Apache Spark](../spark/tutorials/streaming.md) - Spark: spark streaming article
+
+## .NET Standard
+
+### New articles
+
+- [Async Limitations](../standard/data/sqlite/async.md) - Document Microsoft.Data.Sqlite
```

Comment:
```suggestion
- [Async Limitations - Microsoft.Data.Sqlite](../standard/data/sqlite/async.md) - Document Microsoft.Data.Sqlite
```

Reply:
In all cases, we're using the H1 rather than the metadata title for these links. So far, feedback suggests that is a better experience - the link matches the first heading in the article.

Reply:
Shouldn't the h1 then include the "Microsoft.Data.Sqlite" in it ?

Reply:
No, there are other visual cues on each page (TOC, breadcrumb)